src/trace/logfmt.zig

```diff
@@ -32,7 +32,7 @@ fn fieldFmtString(comptime Value: type) []const u8 {
         // Assume arrays of u8 are strings.
         .Pointer => |ptr| if (ptr.size == .One)
             fieldFmtString(ptr.child)
-        else if (ptr.size == .One and ptr.child == u8)
+        else if (ptr.child == u8)
```

Comment:
This is what I was asking about here https://github.com/Syndica/sig/commit/2869887ca94a3ae61e75ed4b2189f12bd8324277#r147741038 having the `ptr.size == .One` in both if and the else looked off.

Nice it got fixed.